test/Git/SerialiseSpec.hs

```diff
@@ -70,6 +83,9 @@ fa7a2abb_uncompBytes = "tree 56558e3275b57381cd04d6cb604dde2f7e773166\n\
     \\n\
     \Fix module name in test\n"
 
+fa7a2abb_loseHeader :: BS.ByteString
```

Comment:
typo loose*

Reply:
(I generally put :+1: reactions to issues I've fixed, for future reference)